cincinnati/src/plugins/internal/graph_builder/openshift_secondary_metadata_parser/plugin.rs

```diff
@@ -295,6 +295,26 @@ impl OpenshiftSecondaryMetadataParserPlugin {
         }
     }
 
+    fn process_version(
+        &self,
+        data_dir: &PathBuf,
+    ) -> Fallible<()> {
+        let supported_version = ["1.0.0"];
+        let path = data_dir.join("version");
+        let version = tokio::fs::read(&path)
+            .await
+            .context(format!("Reading {:?}", &path))?;
+        let trimmed_version = version.trim();
```

Comment:
```suggestion
        let trimmed_version = String::from_utf8_lossy(&version);
```
`fs::read` returns a vector of bytes, it needs to be converted to utf8 string. We shouldn't be trimming here, as it returns a new value - and temp value would be dropped

Reply:
Fixed in c9ef3002 -> 2baa1f2f.